internal/auth/auth.go

```diff
@@ -321,8 +321,9 @@ func (v *verifier) decryptToken() {
 }
 
 func (v verifier) performAuthCheck() (aclResults perms.ACLResults, userId string, scopeInfo *scopes.ScopeInfo, retAcl perms.ACL, retErr error) {
+	const op = "auth.(verifier).performAuthCheck"
 	// Ensure we return an error by default if we forget to set this somewhere
-	retErr = errors.New("unknown")
+	retErr = errors.New(errors.Unknown, op, "")
```

Comment:
Hmmm haven't seen this code before, an interesting way of doing this.  Anyway would be nice to have a clear message even `"default auth error"` to easily trace this since it would be a bug we should fix if it is ever returned

Reply:
"interesting way" lol yes, I had no idea what to put there. thank you!